### PR TITLE
refine logger

### PR DESCRIPTION
**Pull request by binbin0325**
**What this PR does**:
refine logger
**Which issue(s) this PR fixes**:

Fixes https://github.com/arana-db/arana/issues/419

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by dongzl — APPROVED** [review]

**Comment by dongzl** [comment]
How to deal docker?

```
the Dockerfile should contain these environments
```

File: Dockerfile [hunk 1]
```diff
@@ -38,6 +38,13 @@ RUN mkdir ./bin && \
 # runtime layer
 FROM alpine:3
 
+ENV arana_log_name=arana.log \
+    arana_log_level=0 \
+    arana_log_max_size=10 \
+    arana_log_max_backups=5 \
+    arana_log_max_age=30 \
+    arana_log_compress=false
```
Comment (jjeffcaii) [inline Dockerfile:46]: Use upper-case, for example, `ARANA_LOG_LEVEL`.

Comment (binbin0325) [reply Dockerfile]: done

**Review by jjeffcaii — APPROVED** [review]

File: pkg/util/log/logging.go [hunk 2]
```diff
@@ -51,6 +54,24 @@ const (
...
+	// the maximum size in megabytes of the log file before it gets rotated
+	logMaxSizeEnv = "arana_log_max_size"
+	// the maximum number of old log files to retain
+	logMaxBackupsEnv = "arana_log_max_backups"
+	// the maximum number of days to retain old log files
+	logMaxAgeEnv = "arana_log_max_age"
+	// determines if the rotated log files should be compressed using gzip
+	logCompress = "arana_log_compress"
+	// default log name
+	defaultLogName = "arana.log"
```
[the remark this answers is not in the document]
Comment (binbin0325) [reply pkg/util/log/logging.go]: done

**Review by jjeffcaii — APPROVED** [review]

**Pull request merged**